libs/blocks/marquee-container/marquee-container.js

```diff
@@ -0,0 +1,3 @@
+export default function init(el) {
```

Comment:
We already support this in the `loadBlock` function in utils, by adding a `hide-block` class... does anything speak against always appending that class so that we don't need to load empty files?

```
// utils.js
export async function loadBlock(block) {
  if (block.classList.contains('hide-block')) {
    block.remove();
    return null;
  }
```